homeassistant/components/nanoleaf/light.py

```diff
@@ -186,6 +189,10 @@ async def async_update(self) -> None:
         try:
             await self._nanoleaf.get_info()
         except Unavailable:
+            if self.available:
```

Comment:
Does it make sense to use DataUpdateCoordinator instead of implementing this manually?

Reply:
> Does it make sense to use DataUpdateCoordinator instead of implementing this manually?

I don't think it's useful to use a DataUpdateCoordinator for one entity, unless it has other benefits as well?

Reply:
It would likely only be useful if we wanted to do a debounced refresh of the state after calling an api

Example:

https://github.com/home-assistant/core/blob/39d73ecc19179523760738d3943e69198b21167c/homeassistant/components/tplink/entity.py#L18
https://github.com/home-assistant/core/blob/dev/homeassistant/components/tplink/light.py#L92

Reply:
Eventually we will use push updates. So it seems like a lot of work to use a DataUpdateCoordinator for now and then have to change or delete it when we move to push updates. This PR is a minor change so that this integration meets the silver quality scale until we use push updates. Currently, using a data update coordinator does not provide a significant benefit compared to other changes we can make.